docs/adr/0003-lombok.md

```diff
@@ -0,0 +1,54 @@
+# *Use Project Lombok in order to avoid writing boilerplate code*
+
+**User Story:** 
+
+The implemenation of the EffectiveModel must make use of the builder pattern (many optional parameters required for object construction).
```

Comment:
I think using this is only needed in the model, all other pojo style classes are not large enough to need any features of the lombok thing

Reply:
Limiting it to the model might be a good idea.